WooCommerce/Classes/ViewRelated/Dashboard/Free Trial/FreeTrialSurvey/FreeTrialSurveyView.swift

```diff
@@ -0,0 +1,123 @@
+import SwiftUI
+
+final class FreeTrialSurveyHostingController: UIHostingController<FreeTrialSurveyView> {
+    init(viewModel: FreeTrialSurveyViewModel) {
+        super.init(rootView: FreeTrialSurveyView(viewModel: viewModel))
+        rootView.dismissAction = dismiss
+    }
+
+    @available(*, unavailable)
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        configureTransparentNavigationBar()
+    }
+
+    func dismiss() {
+        dismiss(animated: true)
+    }
+}
+
+struct FreeTrialSurveyView: View {
```

Comment:
Nit: Same comment about explaining the type.

Reply:
Done as part of https://github.com/woocommerce/woocommerce-ios/pull/10319/commits/fd5c8beda1db35ee4fd530d76fc6215399cc7b47